src/components/dialogs/parameters/shortcircuit/short-circuit-parameters.tsx

```diff
@@ -156,25 +157,27 @@ const ShortCircuitFields: FunctionComponent<ShortCircuitFieldsProps> = ({ resetA
     return (
         <Grid container spacing={2} paddingLeft={2}>
             <Grid container paddingTop={2} xl={6}>
-                {gridItem(feederResult, 9)}
+                <GridItem size={9}>{feederResult}</GridItem>
             </Grid>
             <GridSection title="ShortCircuitPredefinedParameters" heading={'4'} />
             <Grid xl={6} container spacing={1} alignItems={'center'}>
-                {gridItem(predefinedParameters, 8)}
-                {gridItem(statusToShow, 4)}
+                <GridItem size={8}>{predefinedParameters}</GridItem>
+                <GridItem size={4}>{statusToShow}</GridItem>
             </Grid>
             <GridSection title="ShortCircuitCharacteristics" heading={'4'} />
             <Grid>
-                {gridItem(loads, 6)}
-                {gridItem(shuntCompensators, 6)}
+                <GridItem size={6}>{loads}</GridItem>
+                <GridItem size={6}>{shuntCompensators}</GridItem>
```

Comment:
Isn't 6 the default value?
```suggestion
                <GridItem>{loads}</GridItem>
                <GridItem>{shuntCompensators}</GridItem>
```